web/app/project/settings-dialog/settings-dialog.component.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../common/components/master-detail-card/master-detail-card-shared-styles';
+.fci-detail-container {
+    h3 {
+        font-size: 20px;
+        color: rgba(0, 0, 0, 0.87);
+        font-weight: 500;
+        margin: 0 0 24px 0;
+    }
+    form {
+        display: flex;
+        flex-direction: column;
+        mat-form-field {
```

Comment:
Why is this weirdly highlighted here? GH bug or missing a symbol?

Reply:
it's an unknown element tag. An known element tag would be `form`. But the css rules still work since i believe angular uses web components to represent their components in the DOM.